src/RA_Dlg_AchEditor.h

```diff
@@ -77,22 +71,24 @@ class Dlg_AchievementEditor
     void UpdateCondition(HWND hList, LV_ITEM& item, const Condition& Cond, unsigned int nCurrentHits);
 
 private:
-    static const int m_nNumCols = 10;//;sizeof( g_sColTitles ) / sizeof( g_sColTitles[0] );
+    static constexpr std::size_t m_nNumCols = 10U;
+    static constexpr std::size_t MAX_CONDITIONS = 200U;
+    static constexpr std::size_t MEM_STRING_TEXT_LEN = 80U;
```

Comment:
Just put these all here. I left the `m_nNumCols` name alone, but static class variables aren't really data members since they aren't associated with a class instance.